### PR TITLE
🌱 Bump CAPI to v1.5.5

### PR DESCRIPTION
**Pull request by Sunnatillo**
as title

### CODE REVIEW

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Review by kashifest — CHANGES_REQUESTED** [review]
one comment inline

**Review by kashifest — APPROVED** [review]
/approve

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ need to be manually installed. Example flow of installing Metal3 provider:
    level of the logging verbose with a positive integer number, ie. -v5.
 
    ```shell
-   clusterctl init --core cluster-api:v1.5.3 --bootstrap kubeadm:v1.5.3 \
+   clusterctl init --core cluster-api:v1.5.5 --bootstrap kubeadm:v1.5.3 \
```
[the remark this answers is not in the document]
Comment (Sunnatillo) [reply README.md]: done

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**